envoy-filters/mgw-source/filters/http/mgw-wasm-websocket/filter.cc

```diff
@@ -116,6 +116,7 @@ FilterHeadersStatus MgwWebSocketContext::onResponseHeaders(uint32_t, bool) {
   auto result = getResponseHeaderPairs();
   auto pairs = result->pairs();
   for (auto& p : pairs) {
+    // TODO: (VirajSalaka) Check if wasm API provides an interfact to check if it is an upgrade request.
```

Comment:
There is no specific method to filter out upgrade request.
https://github.com/proxy-wasm/proxy-wasm-cpp-sdk/blob/master/proxy_wasm_api.h